gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -67,14 +68,16 @@ export default function NxButtonPage() {
                           id="nx-button-icon-example"
                           liveExample={NxButtonIconExample}
                           codeExamples={nxButtonIconCode}>
-        An example of a button containing an icon in addition to text.
+          An example of a button containing an icon in addition to text. For accessibility purposes it is important to
+          add an aria-label and aria-hidden=false for screen reader to interpret the content correctly.
       </GalleryExampleTile>
 
       <GalleryExampleTile title="Icon only buttons"
                           id="nx-button-icon-only-example"
                           liveExample={NxButtonIconOnlyExample}
                           codeExamples={nxButtonIconOnlyCode}>
-        An example of a button containing only an icon.
+          An example of a button containing only an icon. For accessibility purposes it is important to add an
+          aria-label and aria-hidden=false for screen reader to interpret the content correctly.
```

Comment:
```suggestion
        An example of a button containing only an icon. For accessibility purposes it is important to add an
        aria-label and aria-hidden=false for screen reader to interpret the content correctly.
```

Reply:
5e921c6360b0259ddb2c519f27d920dbbcf1214d

Reply:
> 5e921c6

That commit does not touch these lines

Reply:
My bad, got the wrong commit, I fixed the indentation in this one 08315e9aebd1b7542a879298c5f28ee0feb7b6dd